.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: cpp
```

Comment:
I don't remember what difference this makes in travis, but is this correct?

Reply:
if you use python it creates a virtualenv that is not used because you're instead installing conda. That's why I usually put c (but in this case it's c++ because of the template...)

Reply:
makes sense